configure.ac

```diff
@@ -45,7 +64,11 @@ fi
 AC_PROG_CC
 AC_PROG_INSTALL
 m4_ifdef([AM_PROG_AR], [AM_PROG_AR])
-LT_INIT
+#if test "$target_os" = "mingw"; then
+	LT_INIT(shared static [win32-dll])
+#else
+#	LT_INIT
+#fi
```

Comment:
Not aware as to LT_INIT is handled my autotools. I think that the conditional check is not required...